packages/node/src/lib/notification-template/notification-template.interface.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  INotificationTemplateStep,
+  IPreferenceChannels,
+} from 'libs/shared/dist';
+
+import { AxiosPromise } from '.pnpm/axios@0.26.1/node_modules/axios';
+
+export interface INotificationTemplates {
```

Comment:
Don't specify the AxiosPromise as a result. So you'll also have to remove the import above